dashboard/apps/prototype/models.py

```diff
@@ -141,13 +152,17 @@ class Project(models.Model):
     project_manager = models.ForeignKey(
         'Person', related_name='projects', null=True)
     client = models.ForeignKey('Client', related_name='projects', null=True)
-    discovery_date = models.DateField(null=True)
-    alpha_date = models.DateField(null=True)
-    beta_date = models.DateField(null=True)
-    live_date = models.DateField(null=True)
-    end_date = models.DateField(null=True)
+    discovery_date = models.DateField(null=True, blank=True)
+    alpha_date = models.DateField(null=True, blank=True)
+    beta_date = models.DateField(null=True, blank=True)
+    live_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+    visible = models.BooleanField(default=True)
     raw_data = JSONField(null=True)
 
+    objects = ProjectManager()
```

Comment:
ignore the comments. we probably don't need to use `all` most of the time, so overriding `objects` is more convenient.